### PR TITLE
User Social Media Controller Test #50

### LINKED ISSUE #50: [Testing] SocialMedia Controller Test
### Resources: 
Resource from Azhya: 
[Spring Boot Testing Basics: How to Unit Test & Integration Test REST Controllers](https://www.youtube.com/watch?v=pNiRNRgi5Ws)

Additional Resources:
[Test Driven Development with Spring Boot - Sannidhi Jalukar, Madhura Bhave](https://www.youtube.com/watch?v=s9vt6UJiHg4)

-----
### Methods: 
As a small reminder, we only need the minimum amount of controller code needed to actually run the test (Calvin's example should show that). 
Note: It's expected to fail. 
Action item: Please go ahead and set up the minimum amount of code (in the Controller layer) for your assigned testing table that you will need to create your test. 

Your tests should have the following methods: 
- getAll
- getById
- Create
- Update
- Delete

### PR DESCRIPTION
**Pull request by briiitelord**
Update - user social media controller and user social media controller test

### CODE REVIEW

File: src/main/java/com/amplifier/controllers/UserSocialMediaController.java [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
 public class UserSocialMediaController {
 
+
+    @Autowired
+    private UserSocialMediaService userSocialMediaService;
+
+    @ApiOperation(value = "Find User Social Media by id number", notes = "Provide an id to lookup a specific user social media from the API", response = UserSocialMedia.class)
+    @GetMapping(path = "/user/user_id/social-media/media_id")
+    public @ResponseBody UserSocialMedia getUserSocialMediaById(@RequestParam(value = "id", name = "media_id") int id) {
+        // System.out.println("TEST: " + userSocialMediaService.getUserSocialMediaById(id));
```
Comment (FullSol) [inline src/main/java/com/amplifier/controllers/UserSocialMediaController.java]: You are knocking these out and I love it!

I am going to remove the extra comments that are not needed, but that's it.

**Review by FullSol — APPROVED** [review]

**Review by pyaeger — APPROVED** [review]
Thank you and approved.

**Comment by pyaeger** [comment]
closes #50

**Comment by FullSol** [comment]
@pyaeger you're good to merge once there are 2

**Pull request merged**